supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -2,7 +2,7 @@
 [[code-examples]]
 = Code examples
 
-Provide the source language if it is supported by the toolchain. Do not use the Bash source language for terminal commands; when representing a terminal prompt for a root command, it improperly interprets the # character as a comment.
+Provide the source language if it is supported by the toolchain. Do not use the Bash source language for terminal commands. Bash improperly interprets the number sign (#) as a comment instead of the prompt for a command run as the root user.
```

Comment:
```suggestion
However, do not use `[source, bash]` for terminal commands. When representing a terminal prompt for a root command, `bash` improperly interprets the # character as a comment.
```
I think you're initial formulation was less ambiguous.

Reply:
I'm fine with either way this was written, but I'd suggest saying `bash` instead of Bash, at least for the first instance.

Reply:
Did some light editing for clarity and implement your suggestion re `bash` in first instance @bergerhoffer